src/plugins/twitter/twitter-fr.hbs

```diff
@@ -20,18 +20,39 @@
 <div lang="en">
 <p><strong>Needs translation</strong></p>
 <section>
-	<h2>Example: Embedded User Timeline</h2>
-	<div class="wb-twitter">
-		<a class="twitter-timeline" href="https://twitter.com/WebExpToolkit" data-tweet-limit="3">Tweets by @WebExpToolkit</a>
-	</div>
+	<h2>Examples: Profile timeline</h2>
+
+	<div class="row">
+		<section class="col-md-6">
+			<h3>Normal account</h3>
+			<div class="wb-twitter">
+				<a class="twitter-timeline" href="https://twitter.com/WebExpToolkit" data-tweet-limit="3">Tweets by @WebExpToolkit</a>
```

Comment:
Indicate to produce a French embedded timeline. We can configure this via `data-lang="fr"`.

```suggestion
				<a class="twitter-timeline" href="https://twitter.com/WebExpToolkit" data-tweet-limit="3" data-lang="fr">Tweets by @WebExpToolkit</a>
```

Reply:
I think I'll take a different approach for this.

Instead of hardcoding ``data-lang``, I'll modify the plugin to automatically inject ``data-lang`` based on the widget's "contextual" language (i.e. parent element or page language). That'll spare implementers from needing to hardcode widget languages.

Btw once the widget loads, the language it displays in doesn't influence tweet or account link destinations (i.e. it won't add a URL param to change Twitter's UI lang to whatever language the widget's UI was using).

Reply:
Marking this as a resolved due to https://github.com/wet-boew/wet-boew/pull/9691#discussion_r1385258973.